pkg/operator/status/condition.go

```diff
@@ -38,7 +40,9 @@ func unionCondition(conditionType string, defaultConditionState operatorv1.Condi
 		return OperatorConditionToClusterOperatorCondition(unionedCondition)
 	}
 
-	if len(badConditions) == 0 {
+	latestTransitionTimeBadConditions := latestTransitionTime(badConditions)
+	if len(badConditions) == 0 || latestTransitionTimeBadConditions.Add(time.Minute).After(time.Now()) {
```

Comment:
Also, we want to gate this on just "Failing" suffix. Yeah, that's ugly, but it's easy and we don't want to delay progressing or available.

Reply:
Done